### DISCUSSION TITLE
Sub-Collections

### DISCUSSION THREAD

**Discussion by saiskee** [outline/outline#1933] [Ideas]
I think that for bigger topics / sections, sub-collections would be really great. If I were to have a collection named "Engineering", it would be helpful to have sub-collections under it like "New Hire Guide", "Product X", "Product Y".

**Comment by tommoor** [comment]
Hi @saiskee, you can nest documents indefinitely – just create a document called "New Hire Guide" and nest others within it

**Comment by mary-loulou** [comment]
@tommoor
I just wanted to give this another shout-out.
Being able to nest collections (sub-collections) would be super helpful.
Documents aren’t a great workaround here — collections provide overview, structure, etc. 
Nested collections would make organising larger wikis a lot easier.

**Comment by mary-loulou** [comment]
@tommoor 
Can you merge the many related requests?
- https://github.com/outline/outline/discussions/9241
- https://github.com/outline/outline/discussions/8805
- https://github.com/outline/outline/discussions/2133
- https://github.com/outline/outline/discussions/1723
